### DISCUSSION TITLE
Gitpod rejects my account

### DISCUSSION THREAD

**Discussion by devvspaces** [fms12/DATA-STRUCTURE-AND-ALGORITHM-IN-PY#203] [Q&A] [answered]
Hi, @fms12 , i tried connecting to gitpod with the link in the readme, but this is what i got
![image](https://user-images.githubusercontent.com/77179231/195796694-07aae86f-d9b4-45f4-baec-f424b3738c0c.png)

**Comment by fms12** [comment]
wait I will check it

**Comment by fms12** [comment, marked as answer]
try now

**Reply by devvspaces** [reply]
Worked thanks, I would happily love to collaborate with you

**Reply by devvspaces** [reply]
Wait, it's still rejecting me

**Comment by fms12** [comment]
got it

**Reply by devvspaces** [reply]
This is the same error `operation not permitted: missing get permission on workspace`
